python/box.tests/build.gradle.kts

```diff
@@ -93,12 +74,7 @@ fun Test.definePythonTestTask(interpreterBinary: String) {
     dependsOn(":kotlin-test:kotlin-test-js-ir:compileKotlinJs")  // todo: remove js stuff
```

Comment:
Is it possible to remove it as well?

Reply:
Not yet: it's related to stdlib and I think it should be handled separately.

> Also see below.

What exactly?

Reply:
Ah, right - I misread something. We talked about it recently on Slack :) All good.